docs/errors.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Errors
+page_title: Errors | Kendo UI Internationalization Package
+description: "List of possible exceptions that the package could throw."
+slug: errors_internalization_kendoui
+position: 6
+---
+
+# Errors list
+
+Below is the list of all controlled errors that could be thrown:
+
+| Type                         | Message
+|:---                          |:---
+|`NoLocaleError`               | Missing locale info for {locale}
```

Comment:
Well, it is "{0}" to be precise - https://github.com/telerik/kendo-intl/blob/exception-handling/src/error-details.js#L5

I wanted to point out that the {locale} is just a placeholder for the current local id. Probably, we could define it like that, no?
```
Missing local info for '[local id here]'
```